### DISCUSSION TITLE
Feature request: Transaction history

### DISCUSSION THREAD

**Discussion by FamousL** [Zrips/CMI#7121] [Feature Requests]
Looking to augment the economy section of the plugin slightly. Currently we can buy/sell/pay, even have admins artificially change the amount of money we have. This feature would allow a user to track changes to their balance even when the transaction happened while they were offline (IE a chest shop was used, or someone sent money while offline)

Not sure if this information is currently saved, making this an easier change, or if a new register would have to be created. This might also help admins in determining where money is going (IE a bunch of players consolidating money to one person might be a cause for concern)

**Comment by mrfloris** [comment]
IF turned on, /money and /sell are logged to log files, check your config.yml.
It would be nice if these transactions are somehow available in-game for the players with perms to see 'last 15 transactions'. 
If third party plugins use transactions, then those plugins have to provide this data, not really something related to cmi who just stores the balance.

**Comment by FamousL** [comment]
Apologies if I was unclear, but yes I was referring to something the users could use without fear to server owners that information would leak unnecessarily. As far as third party plugins,  I am unfamiliar with how the mechanism works. I was under the impression that CMI handled the interface between other plugins and the balance of the users.
